WalletWasabi/WabiSabi/Backend/Banning/CoinVerifierApiClient.cs

```diff
@@ -70,11 +78,14 @@ public virtual async Task<ApiResponseItem> SendRequestAsync(Script script, Cance
 			}
 			catch (Exception ex)
 			{
-				Logger.LogWarning($"API request failed for script: {script}. Remaining tries: {tries}. Exception: {ex}.");
-				await Task.Delay(delay, cancellationToken).ConfigureAwait(false);
+				Logger.LogWarning($"API request failed for script: {script}. Remaining tries: {i}. Exception: {ex}.");
+				await Task.Delay(TimeSpan.FromSeconds(2), cancellationToken).ConfigureAwait(false);
```

Comment:
I know I introduced this delay, but does it make sense to have it if we will use the `SemaphoreSlim`?

It quickly became redundant IMO.

Reply:
Hmm, okay. I'm fine with the delay, I was only worried that it's a leftover from the past.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/10153/commits/d3bc53843519bd0902b9a694540770380b1ee8e4